src/components/PupilViews/PupilLessonOverview/PupilLessonOverview.view.tsx

```diff
@@ -113,6 +113,8 @@ export const PupilViewsLessonOverview = ({
       </OakP>
     ));
 
+  console.log(contentGuidance);
```

Comment:
@weronika-szalas just checking - do we need to keep this console.log or is it just while test the different scenarios?

Reply:
yes pls remove this

Reply:
forgot to remove, done!